modules/autotagging-commons/common/src/main/java/org/opensearch/rule/RuleAttribute.java

```diff
@@ -32,6 +34,11 @@ public String getName() {
         return name;
     }
 
+    @Override
+    public TreeMap<Integer, String> getPrioritizedSubfields() {
+        return new TreeMap<>();
+    }
+
```

Comment:
IMO, this method should be default implement in the `Attribute` interface. Also, can we reuse EMPTY prioritized fields, instead of reinitializing?